github/ci/jenkins-master/tasks/main.yml

```diff
@@ -72,6 +72,11 @@
     port: 8080/tcp
     permanent: true
     state: enabled
+- firewalld:
+    zone: public
+    port: 37947/tcp
```

Comment:
What service does use this port?
And some cosmetic comment(not really need to implement)
you can use `with` statement
```
- firewalld:
     zone: public
     port: "{{ item }}"
     permanent: true
     state: enabled
  with_items:
    - 8080/tcp
    - 37947/tcp
```

Reply:
The swarm plugin started using it for rpc calls.

Reply:
Updated.